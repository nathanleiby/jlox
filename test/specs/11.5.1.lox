fun bad() {
  var a = "first";
  print a;
  var a = "second";
  print a;
}

bad();
