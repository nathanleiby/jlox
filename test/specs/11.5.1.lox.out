Error: Already a variable with name 'a' in this scope.
